domains/POAS/events/attorney-started/schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$id": "https://opg.service.justice.gov.uk/opg.poas.makeregister/attorney-started.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "opg.poas.makeregister/attorney-started",
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "The UID of the LPA",
+      "pattern": "^M(-[A-Z0-9]{4}){3}$"
+    },
+    "actorUID": {
+      "type": "string",
+      "description": "The UID of the attorney",
+      "pattern": "^urn:opg:poas:makeregister:users:([a-z0-9]{8}-)([a-z0-9]{4}-){3}([a-z0-9]{12})$"
+    },
+    "actorType": {
```

Comment:
I'd be tempted to leave this off, you should be able to work it out from the lpa-store data if it is needed for anything